gax/test/showcase-echo-client/package.json

```diff
@@ -35,9 +35,19 @@
     "google-gax": "./google-gax.tgz"
   },
   "devDependencies": {
-    "@types/node": "^20.11.26",
-    "gapic-tools": "./gapic-tools.tgz",
-    "typescript": "^5.7.3"
+    "@types/mocha": "^10.0.7",
```

Comment:
This is a lot of dev dependencies changes - should it be?

Also, we should probably be pointing to local gapic-tools